src/index.js

```diff
@@ -53,6 +53,20 @@ const useStyles = makeStyles((theme) => ({
   root: { overflow: 'visible' },
   table: {
     tableLayout: 'fixed',
+
+    '& thead tr:last-child th:first-child': {
+      backgroundColor: '#FF0000',
+      'border-top-left-radius': 10,
+    },
+
+    '& thead tr:last-child th:last-child': {
+      backgroundColor: '#FF0000',
+      'border-top-right-radius': 10,
+    },
+
+    '& tfoot tr:last-child td:first-child': {
+      'border-bottom': 0,
+    },
```

Comment:
Not a bad idea, but I am worried that it could lead to another styling issue when the container's border radius is different than the hard-coded value. What do you think about dynamically adjust border radius by passing variables? Perhaps we can make its value to be inherited from its parent container

```jsx
  '& thead tr:last-child th:first-child': {
      backgroundColor: '#FF0000',
      'border-top-left-radius':({radius}) => radius,
    },
(...)
const useStyles = styles({radius});
```

Reply:
Also I'd like to mention that the styling issue is only persistent with the table without the top toolbar. For default table form, border should be always square. 
![Screen Shot 2021-04-08 at 12 26 50 PM](https://user-images.githubusercontent.com/59836949/114062613-b6261500-9865-11eb-8f52-8ccc43187b1c.png)

Reply:
yea, it should be dynamically assigned the radius. If there is not prop for round table then it stays square. I will test it how it works